src/containers/Grade/Detail/editTable.jsx

```diff
@@ -69,94 +90,173 @@ export default function EditingGrid({ _columns, _rows }) {
         }
     };
 
-    const processRowUpdate = (newRow) => {
-        const updatedRow = { ...newRow, isNew: false };
+
+    const processRowUpdate = (scoreTypes) => async (newRow) => {
+        const updatedRow = { ...newRow };
+        console.log(updatedRow)
```

Comment:
Use of `console.log` for debugging purposes. Consider removing it before production deployment.

```diff
- console.log(updatedRow)
```